pkgs/servers/ursadb/default.nix

```diff
@@ -33,6 +29,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     zeromq
     cppzmq
+    spdlog
   ];
```

Comment:
All of those are basically noops since UrsaDB links against bundled libraries :thinking: 
It seems like removing those doesn't make any difference.